### PR TITLE
Leaderboard project setup

### PR DESCRIPTION
**Pull request by erickma1**
In this milestone I did the following;

Use Gitflow
Create an npm project with Webpack.
Write plain HTML markup with minimum styling.
Use ES6 modules.

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      template: './src/index.html',
+    }),
+  ],
+  output: {
+    filename: '[name].bundle.js',
+    path: path.resolve(__dirname, 'dist'),
+    clean: true,
+  },
+
+  module: {
```
Comment (LionRouge1) [inline webpack.config.js]: - [ ] Nice work using Webpack for the implementation of the project :+1:  However Webpack **dev server** is not set up in your Webpack configuration. Kindly make sure Webpack **dev server** is set up and running with no errors

File: package.json [hunk 2]
```diff
@@ -2,22 +2,606 @@
   "name": "leaderboard",
   "version": "1.0.0",
   "description": "The leaderboard website displays scores submitted by different players",
-  "main": "index.js",
+  "private": true,
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```
Comment (LionRouge1) [inline package.json]: - [ ] Nice work configuring Webpack command for production mode (`"build": "webpack"`) :clap: However the command for building Webpack in development mode is not configured (`"start": "webpack serve --open"`). Kindly make sure it's configured and the command `npm run start` or `npm run dev` run the **dev server**.

**Review by LionRouge1 — CHANGES_REQUESTED** [review]
Hi @erickma1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Follow [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md) :heavy_check_mark: 
- Use webpack to bundle JavaScript :heavy_check_mark: 
- All linters passed :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by erickma1** [comment]
set up Webpack dev server
configure command for building Webpack in development mode

**Review by OyePriscilla — APPROVED** [review]
Hi @erickma1 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
`Webpack-dev-server` installed ✔️ 
Start app with `npm start` command ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**